configs/zero123.yaml

```diff
@@ -6,23 +6,23 @@ seed: 0
 data_type: "single-image-datamodule"
 data: # threestudio/data/image.py -> SingleImageDataModuleConfig
   image_path: ./load/images/hamburger_rgba.png
-  height: 192
-  width: 192
+  height: 128
+  width: 128
   default_elevation_deg: 0.0
   default_azimuth_deg: 0.0
-  default_camera_distance: 3.2
+  default_camera_distance: 3.8
   default_fovy_deg: 20.0
   random_camera: # threestudio/data/uncond.py -> RandomCameraDataModuleConfig
-    height: 128
-    width: 128
+    height: 64
```

Comment:
I thought you mentioned 96?

Reply:
I was remembering incorrectly when we spoke. ;-)